src/pages/extending-agent-app/index.mdx

```diff
@@ -55,38 +56,44 @@ Additionally, those apps can expose a widget in the native Customer Details tab.
 
 Widgets are simply tiles displayed one below another. This example shows how the **HelpDesk** app extends the default Customer Details tab. Widgets are a great solution for displaying information that agents need to have at hand, as they don't need to switch between tabs.
 
-See the [documentation](/extending-ui/extending-agent-app/agent-app-sdk/#getting-started) to learn how to embedd your app in the Details section.
+See the documentation and learn how to set up [LiveChat Widgets](/extending-agent-app/products-sdk/agent-app-sdk/#getting-started) or [HelpDesk Widgets](/extending-agent-app/products-sdk/helpdesk-sdk/#getting-started), including the ones in the Details section.
 
-## Main menu (fullscreen apps)
+## Fullscreen App
 
 The application icon is located in the **navigation bar** on the left.
 
-Clicking the icon loads the app URL as an iframe. The application is displayed fullscreen, right in the LiveChat App.
+Clicking the icon loads the app URL as an iframe. The application is displayed fullscreen, right in the LiveChat or HelpDesk App.
 
 <div style={{marginBottom: "15px"}}>
 <img alt="LiveChat Fullscreen app" src="/images/extending-agent-app/livechat-fullscreen-app.jpg" width="780px" height="490px"/>
 </div>
 
 Keep in mind that this feature is not supported on the mobile version of the LiveChat Agent App.
 
-Fullscreen apps help to reduce _context switching_. Users no longer need to switch between the Agent App and tabs in the browser. As long as the application they work with is installed as a **Fullscreen app**, they can perform all the operations without leaving the Agent App. It's particularly important for LiveChat Desktop App users.
+Fullscreen apps help to reduce _context switching_. Users no longer need to switch between the Agent App and tabs in the browser. As long as the application they work with is installed as a **Fullscreen App**, they can perform all the operations without leaving the Agent App. It's particularly important for LiveChat Desktop App users.
 
-See how to create a fullscreen app in the [documentation](/extending-ui/extending-agent-app/agent-app-sdk/#fullscreen-widgets).
+See the documentation and learn how to set up a fullscreen app in the [LiveChat App](/extending-agent-app/products-sdk/agent-app-sdk#fullscreen-widgets) or [HelpDesk App](/extending-agent-app/products-sdk/helpdesk-sdk#fullscreen-widgets).
 
-## MessageBox (available only for LiveChat)
+## Message Box (available only for LiveChat)
 
 With this type of widget placement you can integrate your apps in the Message Box. You could use it to send gifs or present items from an online store. It would make the whole chatting experience much more engaging. You could also use it to send files from [Dropbox](https://www.livechat.com/marketplace/apps/dropbox/).
```

Comment:
```suggestion
With this type of widget placement you can integrate your apps in the message box area. You could use it to send gifs or present items from an online store. It would make the whole chatting experience much more engaging. You could also use it to send files from [Dropbox](https://www.livechat.com/marketplace/apps/dropbox/).
```